packages/wesl-ext-typegpu/src/index.ts

```diff
@@ -3,12 +3,18 @@ import type { PluginExtensionApi } from 'wesl-plugin';
 import { generateImportSnippets } from './imports.ts';
 import { generateStructSnippets } from './structs.ts';
 
-export const typegpuExtension = {
+export const typegpuExtension: {
+  extensionName: 'typegpu';
+  emitFn: typeof emitReflectJs;
+} = {
   extensionName: 'typegpu',
   emitFn: emitReflectJs,
 };
```

Comment:
Can we use `as const` here?
```suggestion
export const typegpuExtension = {
  extensionName: 'typegpu',
  emitFn: emitReflectJs,
} as const;
```

Reply:
I tried that, but it didn't work 😭. Seems like it's very fussy about having the types be explicit.